action.yml

```
@@ -85,14 +85,14 @@ runs:
 
         if [ -z "$TERMINUS_USER" ]; then
           echo "No valid session found. "
-          exit 1
+          exit 0
```

```suggestion
          exit
```
it's gonna exit with echo's exit code so the 0 is redundant.

---

I'd recommend leaving it 0 for explicitness.